bcipy/gui/viewer/data_viewer.py

```diff
@@ -318,6 +318,9 @@ def update_buffer(self, fast_forward=False):
                 self.buffer.append(row)
         except StopIteration:
             self.stop()
+            self.Close()
```

Comment:
Is this closing Wx.Frame? We could have a `close` or `exit` method that wraps this which might be more clear